s3torchbenchmarking/conf/dataloading.yaml

```diff
@@ -9,5 +9,5 @@ hydra:
   mode: MULTIRUN
   sweeper:
     params:
-      dataloader: s3iterabledataset, fsspec
+      dataloader: s3iterabledataset, fsspec, mountpoint
```

Comment:
This is the default right? Do we want to have everyone running benchmarking to need MP installed?

Reply:
Agreed. This should be opt-in rather than enforced.